dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,156 @@
+
+# "URL for ACME CA account: " + account.getLocation());
+CWPKI2019I=CWPKI2019I: The account URL provided by the ACME certificate authority at URI {0} is {1}.
+CWPKI2019I.explanation=The account was successfully created.
+CWPKI2019I.useraction=No action is required.
+
+#"Error reading domain key file."
+CWPKI2020E=CWPKI2020E: Failed to read the domain key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2020E.explanation=The domain key file for the certificate authority account could not be opened.
+CWPKI2020E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+#"Error reading account key file."
+CWPKI2021E=CWPKI2021E: Failed to read the account key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2021E.explanation=The account key file for the certificate authority account could not be opened.
+CWPKI2021E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+#"Error writing domain key file."
+CWPKI2022E=CWPKI2022E: Failed to write to the domain key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2022E.explanation=Could not write to the domain key file for the certificate authority account.
+CWPKI2022E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+#"Error writing account key file."
+CWPKI2023E=CWPKI2023E: Failed to write the account key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2023E.explanation=Could not write to the account key file for the certificate authority account.
+CWPKI2023E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+# "Error revoking certificate."
+CWPKI2024E=CWPKI2024E: Failed to revoke the requested certificate for the ACME certificate authority at URI {0}. The certificate that failed to revoke has serial number {1}. The error is {2}.
+CWPKI2024E.explanation=A revoke request failed. The certificate may not be revoked.
+CWPKI2024E.useraction=Review the provided exception for details on the failure.
+
+#"Could not load account KeyPair. Revoking a certificate requires a valid account key pair."
+CWPKI2025W=CWPKI2025W: Received a certificate revoke request, but the account key pair for the ACME certificate authority at URI {0} could not be loaded.
+CWPKI2025W.explanation=Could not revoke a certificate because the account key pair could not be loaded.
+CWPKI2025W.useraction=Review the log for any earlier exceptions for details on the failure. Or manually review the key store file.
```

Comment:
Or manually review the key store file.

is this to check for errors in that file config?